### ISSUE TITLE
Training DCNv2 on ImageNet-1K

### ISSUE THREAD

**Issue by ToTheBeginning** [open-mmlab/mmdetection#530]
I found it very difficult to train DCNv2 on ImageNet, for example:

1. After first epoch, DCNv2 can only achieve an avg top1(top5) accuracy of 0.9%(3.38%) on training set, and avg top1(top5) accuracy of **0.1%(0.5%)** on val set. I built the model exactly according to your code, the initialization of the model parameters was also taken into account. The code for training and testing is also unlikely to go wrong, because the result is normal after replacing DCNv2 with resnet.

2. The loss of validation is very huge

So, do you have any suggestions for me, such as the training tricks not mentioned in the paper? That will be very helpful!

**Comment by hellock** [comment]
Sorry that we have not trained DCNv2 on ImageNet, maybe you need to ask for the authors' help.

**Comment by ToTheBeginning** [comment]
OK, thank you. After 5 epoch, the loss and accuracy tend to be normal.

But the DCNv2 is so slow... e.g., DCN50 is about 5 times slower than resnet50.

**Comment by shivin9** [comment]
Hi @ToTheBeginning,

Can you share your implementation? I am not understanding how the pooling operation works for DCN's.

Thanks,
Shivin

**Comment by vanpersie32** [comment]
@ToTheBeginning Have you trained the DCN on imagenet? My loss stay the same and don't decrease

**Comment by ToTheBeginning** [comment]
@vanpersie32 Yes, I train it for a while, the loss can decrease, but the test loss needs to wait for a few epoch to drop. But I found it too slow to train even in a 8-cards environment, so I just give up training.
